mandatory/2-logic-error.js

```diff
@@ -1,16 +1,17 @@
 // The syntax for these functions is valid but there are some errors, find them and fix them
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length();
 }
 
-function multiply(a, b, c) {
-  a * b * c;
-  return;
+function multiply(a, b, c) 
+{
+  return; a * b * c
```

Comment:
It is worth double checking this line.
Running the tests will help you find out if your code is working.